src/pages/instances/InstanceList.tsx

```diff
@@ -69,6 +70,10 @@ const InstanceList: FC = () => {
     return <>Missing project</>;
   }
 
+  if (!location.pathname.endsWith("/instances")) {
```

Comment:
I think we should not use the InstanceList as a redirector. It is a different functionality.

Reply:
Any alternative idea then?